### ISSUE TITLE
Adding output with action result

### ISSUE THREAD

**Issue by karpikpl** [thehanimo/pr-title-checker#50]
I think having output variable with the check results would be useful to chain more actions, e.g. add a PR comment.

I could do a PR for it if you're open to having this feature.

**Comment by thehanimo** [comment]
Would this be a single Boolean? Or something like #30?

**Comment by karpikpl** [comment]
I was thinking about Boolean, I saw #30 and thought it might not be that easy to handle.
In the workflow I was doing, I wanted to add a comment with explanation on what titles are accepted.

Triggering of label change doesn't work when `GITHUB_TOKEN` is used, so a simple `true/false` result would help

**Comment by thehanimo** [comment]
Sure, if you could create a PR that'd be great. This should be backwards compatible right?

**Comment by karpikpl** [comment]
that's right, using outputs is optional and doesn't change how action is executed
